### PR TITLE
Ashwat's pull request

### PR DESCRIPTION
**Pull request by iamaks287**
bubble sort in c++

### CODE REVIEW

File: c++/bubble_sort.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+using namespace std;
+
+void bubbleSort(int arr[], int s) 
+{
+  for (int step = 0; step < (s-1); ++step) 
+  {
+    for (int i = 0; i < s - (step-1); ++i)
```
Comment (king-11) [inline c++/bubble_sort.cpp:8]: This is doesn't seem right

at `step = 0`, `i` will go until `s` which is array out of bounds it should be `s - step - 1`

File: c++/bubble_sort.cpp [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+using namespace std;
+
+void bubbleSort(int arr[], int s)
```
Comment (king-11) [inline c++/bubble_sort.cpp:4]: Move function definition after main just keep the function declaration here.

**Review by king-11 — CHANGES_REQUESTED** [review]

**Comment by iamaks287** [comment]
I have made the suggested changes

**Review by king-11 — APPROVED** [review]

**Comment by king-11** [comment]
Have you tested your program this time?

**Comment by iamaks287** [comment]
Yeah i have tested it and it's working fine

**Pull request merged**